folder.rb

```diff
@@ -0,0 +1,62 @@
+require 'fileutils'
+
+class Folder
+
+  def self.check_folder(folder)
+
+    return false if !File.directory?(folder)
+
+    folders = Dir.entries(folder)
+
+    if check_important(folders).inspect == 'true'
```

Comment:
```suggestion
    return true if check_important(folders)
```

Can you please refactor this line to use a guard clause ? Also the method `check_important` returns `true`  or `nil` it doesn't seems necessary to use `.inspect` method